external/multi-base-images/flink/flink-entrypoint.sh

```diff
@@ -42,6 +42,8 @@ fi
 echo "web.upload.dir: $FLINK_HOME" >> "$FLINK_HOME/conf/flink-conf.yaml"
 echo "jobmanager.web.upload.dir: $FLINK_HOME" >> "$FLINK_HOME/conf/flink-conf.yaml"
 
+echo "taskmanager.memory.flink.size: 1024mb" >> "$FLINK_HOME/conf/flink-conf.yaml"
```

Comment:
A larger default needs to be provided. Without this line, the following error will occur:
```
 - 'taskmanager.memory.flink.size' is not specified, use the configured deprecated task manager heap value (512.000mb (536870912 bytes)) for it.
 - The derived from fraction network memory (51.200mb (53687092 bytes)) is less than its min value 64.000mb (67108864 bytes), min value will be used instead
Exception in thread "main" org.apache.flink.configuration.IllegalConfigurationException: Sum of configured Framework Heap Memory (128.000mb (134217728 bytes)), Framework Off-Heap Memory (128.000mb (134217728 bytes)), Task Off-Heap Memory (0 bytes), Managed Memory (204.800mb (214748368 bytes)) and Network Memory (64.000mb (67108864 bytes)) exceed configured Total Flink Memory (512.000mb (536870912 bytes)).
	at org.apache.flink.runtime.clusterframework.TaskExecutorProcessUtils.deriveInternalMemoryFromTotalFlinkMemory(TaskExecutorProcessUtils.java:320)
	at org.apache.flink.runtime.clusterframework.TaskExecutorProcessUtils.deriveProcessSpecWithTotalFlinkMemory(TaskExecutorProcessUtils.java:221)
	at org.apache.flink.runtime.clusterframework.TaskExecutorProcessUtils.processSpecFromConfig(TaskExecutorProcessUtils.java:143)
	at org.apache.flink.runtime.util.BashJavaUtils.getTmResourceJvmParams(BashJavaUtils.java:62)
	at org.apache.flink.runtime.util.BashJavaUtils.main(BashJavaUtils.java:46)
[ERROR] Could not get JVM parameters properly.
```